cytominer_database/munge.py

```diff
@@ -0,0 +1,65 @@
+"""
+
+"""
+
+import click
+import cytominer_database.utils
+import os
+import pandas as pd
+
+
+def munge(config, source, target=None):
+    """
```

Comment:
Can you include a description of what this function does to each CSV file it finds?